src/components/TagsBox.vue

```diff
@@ -2,9 +2,9 @@
 import { GridItem, GridLayout } from 'vue3-drr-grid-layout'
 import 'vue3-drr-grid-layout/dist/style.css'
 import { nextTick } from 'vue'
-
 import Kxz from '@/assets/ico/kxz.png'
 
+// 123545678
```

Comment:
These comments seem to be unnecessary information. Deleting them can help us improve the quality of the code.